snuba/web/views.py

```diff
@@ -431,11 +431,23 @@ def snql_dataset_query_view(*, dataset: Dataset, timer: Timer) -> Union[Response
 # https://github.com/ClickHouse/ClickHouse/blob/1c0b731ea6b86ce3bf7f88bd3ec27df7b218454d/src/Common/ErrorCodes.cpp
 ILLEGAL_TYPE_OF_ARGUMENT = 43
 TYPE_MISMATCH = 53
+
+# queries that can never return the amount of data requested by the user are not an internal error
```

Comment:
can we put these constants into their file along with a function like:

```python3
def get_http_status_for_clickhouse_error(error: ClickhoueError) -> int:
    pass
```

This file has too much stuff in it already